bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -423,6 +423,13 @@ void ReplicaImp::startConsensusProcess(PrePrepareMsg *pp, bool isInternalNoop) {
 }
 
 void ReplicaImp::sendInternalNoopPrePrepareMsg(CommitPath firstPath) {
+  if (primaryLastUsedSeqNum + 1 > lastStableSeqNum + kWorkWindowSize) {
+    LOG_INFO(GL,
```

Comment:
Does this need to be an INFO message? Why not make it DEBUG?
Additionally, since we are already logging noops with the `CNSUS` logger, we should probably use that.